ci/test-checks.sh

```diff
@@ -13,7 +13,7 @@ export RUSTFLAGS="-D warnings"
 # Look for failed mergify.io backports
 _ git show HEAD --check --oneline
 
-if _ scripts/cargo-for-all-lock-files.sh check --locked; then
+if _ scripts/cargo-for-all-lock-files.sh +"$rust_nightly" check --locked --all-targets; then
```

Comment:
`+"$rust_nightly"` needed for this: https://buildkite.com/solana-labs/solana/builds/23350#c4cb3312-b32d-42b1-bda2-1e2cdc091a4b/70-475

```
    Checking solana-ownable v1.2.0 (/solana/programs/ownable)
    Checking solana-btc-spv-program v1.2.0 (/solana/programs/btc_spv)
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> sdk/benches/slot_history.rs:1:1
  |
1 | #![feature(test)]
  | ^^^^^^^^^^^^^^^^^
 
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> sdk/benches/slot_hashes.rs:1:1
  |
1 | #![feature(test)]
  | ^^^^^^^^^^^^^^^^^
 
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> sdk/benches/short_vec.rs:1:1
  |
1 | #![feature(test)]
  | ^^^^^^^^^^^^^^^^^
 
error: aborting due to previous error
```